homework9/my-app/src/App.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
```

Comment:
you don't need a folder `my-app`
should be as `homework9/project` but now u have as `homework9/my-app/project`

Reply:
I did not create it myself. I came up with "create react app".
Should I cut all my files from my-app and paste into homework9?

Reply:
Yes. All files should be inside homework9
you ran this command in the Homework 9 folder